src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -64,7 +64,7 @@ static function ($deferred, $namespace, &$expiredIds, $getId, $defaultLifetime)
                 foreach ($deferred as $key => $item) {
                     $key = (string) $key;
                     if (null === $item->expiry) {
-                        $ttl = 0 < $defaultLifetime ? $defaultLifetime : 0;
+                        $ttl = max(0, $defaultLifetime);
```

Comment:
This is mostly a readability improvement and I agree with switching to `max()` in most of the places where you've applied this change.

However, the ternary expression might be a tiny bit faster than a function call. Given that this particular piece code might be executed on a hot path, I'd revert this kind of change on the cache adapters.

Reply:
Thanks. This is now reverted.